lib/js/components/Divider/Divider.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<div
+		class="divider"
+		:class="{
+			'-vertical': isVertical,
+			'-horizontal': !isVertical,
+			'-strong': prominence === PROMINENCES.STRONG,
+			'-weak': prominence === PROMINENCES.WEAK,
+		}"
+	/>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors/tokens';
+
+.divider {
+	background-color: $color-neutral-divider;
+
+	&.-strong {
+		background-color: $color-neutral-divider-strong;
+	}
+
+	&.-weak {
+		background-color: $color-neutral-divider-weak;
+	}
+
+	&.-vertical {
+		height: 100%;
+		width: 1px;
+	}
+
+	&.-horizontal {
+		height: 1px;
+		width: 100%;
+	}
+}
+</style>
+
+<script lang="ts">
+import { PROMINENCES } from './Divider.consts';
+
+export default {
+	name: 'Divider',
+	props: {
+		isVertical: {
+			type: Boolean,
+			required: true,
```

Comment:
According to https://miro.com/app/board/uXjVO_rTPmE=/?moveToWidget=3458764522565713964&cot=10, this should be optional